frappe/custom/doctype/package_publish_tool/package_publish_tool.py

```diff
@@ -110,7 +110,7 @@ def import_package(package=None):
 		exists = frappe.db.exists(doc.get("doctype"), doc.get("name"))
```

Comment:
I think we should add `frappe.only_for("System Manager")` at the top of the method